cpp/include/cudf/aggregation.hpp

```diff
@@ -34,6 +34,13 @@
 
 namespace cudf {
 namespace experimental {
+/**
+ * @brief Aggregation APIs
+ * @ingroup column_apis
+ * @addtogroup column_aggregation Aggregation
```

Comment:
These are not just column aggregations, right?

```suggestion
 * @addtogroup column_aggregation Aggregation
```

Reply:
> These are not just column aggregations, right?

What do you suggest? Should I make them their own group then?